src/sidebar/util/test/search-filter-test.js

```diff
@@ -185,7 +224,7 @@ describe('sidebar/util/search-filter', () => {
     });
 
     it('filters to a focused user', () => {
-      const filter = searchFilter.generateFacetedFilter(null, {
+      const filter = searchFilter.generateFacetedFilter('', {
```

Comment:
Per the type definition of `generateFacetedFilter`, the first argument is required to be a string. `null` happened to work previously. App code always passes a (possibly empty) string as the first arg.